core/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -408,7 +408,9 @@ export default class DAppWrapperComponent extends mixins(EvanComponent) {
    * runtimes
    */
   async handleLoginOnboarding(): Promise<void> {
-    this.stopSessionWatcher = await session.start(async (action: string) => {
+    this.stopSessionWatcher = await session.start(async (action: string): Promise<any> => {
+      let result;
```

Comment:
Can we improve this somehow? Having a return value in only a specific case? Seems like code smell if we need to cast `any`.

Reply:
Will be only string or null, changed that to string return.

Reply:
The compiler doesn't complain about this? It can be either undefined (not null!) or string now. That's surprising.

Reply:
Sublime and not even VSCode complains about it :D
You are my better compiler, what do you think it should be like?

Reply:
compiler probably doesn't complain, bc anon function in L452 isn't typed.